### PR TITLE
Phase1branch2

### PR DESCRIPTION
**Pull request by jhelphenstine**
This is a pull request to bring the Phase1branch2 code (mostly database stuff) into the phase1 branch where the main work has occurred on the ui, preventing us from trying to keep these 2 branches updated.

### CODE REVIEW

File: src/main/java/cmsc495/ui/Page_BrowseRecipe.java [hunk 1]
```diff
@@ -114,7 +119,7 @@ public void actionPerformed(ActionEvent e) {
         
         // set the panel to the main page
         SimpleGui gui = (SimpleGui)SwingUtilities.getRoot(button);
-        gui.setCurrentPage(new Page_DisplayRecipe(recipe));
+        //gui.setCurrentPage(new Page_DisplayRecipe(recipe));
```
Comment (eliotpearson) [inline src/main/java/cmsc495/ui/Page_BrowseRecipe.java:118]: Had to comment this out.  The would create a compile time failure.

**Review by eliotpearson — APPROVED** [review]
Changes look good to me, but someone else should review it as well.

File: src/main/java/cmsc495/ui/Page_BrowseRecipe.java (hunk 1 above)
Comment (ahowell82) [reply src/main/java/cmsc495/ui/Page_BrowseRecipe.java:118]: What is the "<<<<<<<<HEAD....." extra stuff??

Comment (ahowell82) [reply src/main/java/cmsc495/ui/Page_BrowseRecipe.java:118]: Line should not create a compile time error ... what is the error?

**Review by ahowell82 — APPROVED** [review]
Gui Team will fix whatever gets blown away (if any exists)

File: src/main/java/cmsc495/ui/Page_BrowseRecipe.java (hunk 1 above)
Comment (eliotpearson) [reply src/main/java/cmsc495/ui/Page_BrowseRecipe.java:118]: Fixed <<<<<HEAD issue.  Looking into compile time error.

Comment (eliotpearson) [reply src/main/java/cmsc495/ui/Page_BrowseRecipe.java:118]: @ahowell82 When I uncomment link 122.  I receive, "The constructor Page_DisplayRecipe(Recipe_Hold) is undefined."  Double checking my merge...

**Pull request merged**